pkg/sources/git/git.go

```diff
@@ -147,7 +174,7 @@ func (s *Source) Chunks(ctx context.Context, chunksChan chan *sources.Chunk) err
 				if err != nil {
 					return err
 				}
-				return s.git.ScanRepo(ctx, repo, path, NewScanOptions(), chunksChan)
+				return s.git.ScanRepo(ctx, repo, path, s.scanOptions, chunksChan)
```

Comment:
I'm not initializing `s.scanOptions`, so the default value of `nil` will be used if it's not set. This is okay because the `ScanRepo` function already catches this:

https://github.com/trufflesecurity/trufflehog/blob/8ec5e4916c911144aab39e0602dfa930d3aa2d09/pkg/sources/git/git.go#L562-L565